<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllAlbumsWarningDialog</class>
 <widget class="QDialog" name="AllAlbumsWarningDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
       <width>696</width>
       <height>604</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Warnings: different embedded covers</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QLabel" name="iconLabel">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/icons/dialog-warning.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="headerLabel">
       <property name="text">
        <string>Albums with differing embedded covers</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
       <widget class="QTabWidget" name="tabWidget">
           <widget class="QWidget" name="tab_with_cover">
               <attribute name="title">
                   <string>with a cover</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                       <widget class="QTextBrowser" name="browser">
                           <property name="readOnly">
                               <bool>true</bool>
                           </property>
                           <property name="openExternalLinks">
                               <bool>false</bool>
                           </property>
                       </widget>
                   </item>
               </layout>
           </widget>
           <widget class="QWidget" name="tab_without_cover">
               <attribute name="title">
                   <string>without a cover</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                       <widget class="QTextBrowser" name="browser_no_cover">
                           <property name="readOnly">
                               <bool>true</bool>
                           </property>
                           <property name="openExternalLinks">
                               <bool>false</bool>
                           </property>
                       </widget>
                   </item>
               </layout>
           </widget>
           <widget class="QWidget" name="tab_errors">
               <attribute name="title">
                   <string>errors</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item>
                       <widget class="QTextBrowser" name="browser_errors">
                           <property name="readOnly">
                               <bool>true</bool>
                           </property>
                           <property name="openExternalLinks">
                               <bool>false</bool>
                           </property>
                       </widget>
                   </item>
               </layout>
           </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
